app/public/src/pages/component/synergy/synergy-detail-component.tsx

```diff
@@ -48,11 +48,21 @@ export default function SynergyDetailComponent(props: {
     )
     .map((p) => getPokemonData(p as Pkm))
     .sort((a, b) => RarityCost[a.rarity] - RarityCost[b.rarity])
+  
+  const baseVariant = (basePkm: Pkm): Pkm => {
```

Comment:
the naming of variables is weird, i suggest:

```suggestion
  const baseVariant = (pkm: Pkm): Pkm =>  (Object.keys(PkmRegionalVariants) as Pkm[]).find((p) =>
      PkmRegionalVariants[p]!.includes(pkm)
    ) ?? pkm
```